devimint/src/external.rs

```diff
@@ -373,6 +374,60 @@ pub async fn open_channel(bitcoind: &Bitcoind, cln: &Lightningd, lnd: &Lnd) -> R
     })
     .await?;
     bitcoind.mine_blocks(10).await?;
+
+    poll("Wait for channel update", || async {
+        let mut lnd_client = lnd.client.lock().await;
+        let channels = lnd_client
+            .lightning()
+            .list_channels(ListChannelsRequest {
+                active_only: true,
+                ..Default::default()
+            })
+            .await?
+            .into_inner();
+
+        if channels.channels.is_empty() {
+            return Ok(false);
+        }
+
+        let chan_id = channels
+            .channels
+            .get(0)
```

Comment:
Not waiting for all channels could lead to flakiness if we ever open more because sometimes the other channels might be ready and sometimes not, leading to `update_channel_policy` taking effect on them or not.

Reply:
I reworked this a bit, now it explicitly waits for a channel with the `remote_public_key` that corresponds to CLN. That should help prevent flakiness in the future